x/metrix/keeper/keeper.go

```diff
@@ -344,11 +344,11 @@ func (k *Keeper) UpdateRelayMetrics(ctx context.Context) {
 	logger.Debug("Updating relay metrics finished!")
 }
 
-func (k *Keeper) UpdateUptime(ctx context.Context) error {
+func (k *Keeper) UpdateUptime(ctx context.Context) {
 	sdkCtx := sdk.UnwrapSDKContext(ctx)
 	window, err := k.slashing.SignedBlocksWindow(ctx)
 	if err != nil {
-		return err
+		k.Logger(sdkCtx).With("Err", err).Error("Error while getting the SignedBlocksWindow")
```

Comment:
What did we learn about the difference of doing this vs. using liblog?

Reply:
`K.Logger` and  `liblog` are different .They have different usage.